.babelrc

```diff
@@ -1,14 +1,19 @@
 {
   "presets": [
-    "@babel/preset-react",
     [
       "@babel/preset-env",
       {
         "targets": {
           "browsers": ["last 2 Chrome versions", "last 2 Firefox versions"]
         }
       }
-    ]
+    ],
+    "@babel/preset-react",
+    "@babel/preset-typescript"
   ],
-  "plugins": ["@babel/plugin-proposal-class-properties", "@babel/plugin-proposal-optional-chaining"]
+  "plugins": [
+    "@babel/plugin-proposal-class-properties", 
+    "@babel/plugin-proposal-optional-chaining", 
+    ["react-remove-properties", {"properties": ["data-testid"]}]
```

Comment:
We'll be making use of `react-testing-library` and it's nifty [`byTestId` query](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid) to unit test our React components. This plugin removes those attributes during the build process to avoid them showing up in production.